egs/himia/wuw/ctc_tdnn/graph.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# Copyright    2023  Xiaomi Corp.        (Author: Weiji Zhuang,
+#                                                 Liyong Guo)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+
+def ctc_trivial_decoding_graph(wakeup_word_tokens: List[int]):
+    """
+    A graph starts with blank/unknown and following by wakeup word.
+
+    Args:
+      wakeup_word_tokens: A sequence of token ids corresponding wakeup_word.
+      It should not contain 0 and 1.
+      We assume 0 is for blank and 1 is for unknown.
```

Comment:
Could you also document the return value?

Reply:
fixed. Thanks.